image/cli/mascli/functions/update

```diff
@@ -146,6 +146,9 @@ function validate_dro_migration(){
         fi
 
         if [[ "$DRO_MIGRATION" == "true" ]]; then
+          if prompt_for_confirm_default_yes "Do you want to use your own storage class to configure DRO?"; then
```

Comment:
we need to support Non-interactive install for this prompt as well.

Reply:
use `if [[ "$NO_CONFIRM" != "true" ]];` then like i used here. 

![image](https://github.com/ibm-mas/cli/assets/110647904/cc4e29b3-ecb9-4b79-b335-2652972b889d)